github/git_commits.go

```diff
@@ -35,7 +36,8 @@ type Commit struct {
 	// CommentCount is the number of GitHub comments on the commit. This
 	// is only populated for requests that fetch GitHub data like
 	// Pulls.ListCommits, Repositories.ListCommits, etc.
-	CommentCount *int `json:"comment_count,omitempty"`
+	CommentCount *int    `json:"comment_count,omitempty"`
+	NodeID       *string `json:"node_id,omitempty"`
```

Comment:
I think it'd be nicer to either move `NodeID` field up, together with all other fields, or add a blank line after `CommentCount`, since it has a comment saying that "This is only populated for requests that fetch GitHub data", which doesn't apply to `NodeID` as I understand.

Minor, but worth doing.